pages/features.html

```diff
@@ -100,7 +100,7 @@ <h2>{% trans "MIDI and HID Controller Support" %}</h2>
 
     <h2>{% trans "Free Timecode Vinyl Control" %}</h2>
     <p>
-      {% blocktrans with vinyl_control_manual_link="/manual/latest/chapters/vinyl_control.html" %}Control your digital music files from turntables or CDJs and a mixer using timecoded vinyl or CDs. Grab the music as if it was pressed onto wax and scratch to your heart's content. Mixxx is the only free timecode vinyl control software for Windows, macOS, and Linux. <a href="{{ vinyl_control_manual_link }}">Read more</a> {% endblocktrans %}
+      {% blocktrans with vinyl_control_manual_link="/manual/latest/en/chapters/vinyl_control.html" %}Control your digital music files from turntables or CDJs and a mixer using timecoded vinyl or CDs. Grab the music as if it was pressed onto wax and scratch to your heart's content. Mixxx is the only free timecode vinyl control software for Windows, macOS, and Linux. <a href="{{ vinyl_control_manual_link }}">Read more</a> {% endblocktrans %}
```

Comment:
Doesn't hardcoding the link to the English version of the manual bypass the automatic redirection to the localized manual?

Reply:
Yup, a rewrite would be better.

Reply:
Please check mixxxdj/manual#102